setup.cfg

```diff
@@ -1,2 +1,6 @@
 [flake8]
-ignore=E501
+# Ignore:
+#
+#   N812 - We like doing EC for the expected_conditions, even 
```

Comment:
- What is EC?
- I like the idea of adding a short description for errors that are being ignored. We'll need to add one for E501